account_payment_return_import_sepa_pain/tests/test_import_sepa_pain.py

```diff
@@ -20,6 +20,7 @@ def setUp(self):
             'bank_name': 'TEST BANK',
             'company_id': self.company.partner_id.id,
         })
+        self.acc_bank.journal_id.payment_return_pattern = '.*'
```

Comment:
Please elaborate a more complicated pattern and example to check that this works.